src/ast/mod.rs

```diff
@@ -6925,6 +6928,12 @@ pub enum SqlOption {
         range_direction: Option<PartitionRangeDirection>,
         for_values: Vec<Expr>,
     },
+
+    TableSpace(TablespaceOption),
+
+    Union(Vec<Ident>),
+
+    TableEngine(TableEngine),
```

Comment:
Could we add a description (ideally with syntax example) and links to the docs where these variants come from?